examples/taskruns/secret-volume-params.yaml

```diff
@@ -33,4 +33,4 @@ spec:
   inputs:
     params:
     - name: SCNAME
```

Comment:
I think we need to have the `type` here… This feels weird to me but looking at the error…
```
I1009 13:16:09.670] Error from server (InternalError): error when creating "STDIN": Internal error occurred: admission webhook "webhook.tekton.dev" denied the request: mutation failed: invalid value: : taskspec.inputs.params.SCNAME.type
```

Reply:
Nice catch. It may be because it's in a `taskSpec` embedded in a `TaskRun`.
I fixed this and all other occurrences.